pkg/resources/cloudcontroller/openstack.go

```diff
@@ -171,13 +171,19 @@ func getOSFlags(data *resources.TemplateData) []string {
 }
 
 func getOSVersion(version semver.Semver) (string, error) {
+	if version.Major() < 17 {
+		return "", fmt.Errorf("Kubernetes version %s is not supported", version.String())
+	}
+
 	switch version.Minor() {
 	case 17:
 		return "1.17.0", nil
 	case 18:
 		return "1.18.0", nil
+	case 19:
+		return "1.19.3", nil
```

Comment:
I don't think this version exists yet:
```
curl https://registry.hub.docker.com/v1/repositories/k8scloudprovider/openstack-cloud-controller-manager/tags
[{"layer": "", "name": "latest"}, {"layer": "", "name": "1.13.0"}, {"layer": "", "name": "1.13.1"}, {"layer": "", "name": "v0.1.0"}, {"layer": "", "name": "v0.2.0"}, {"layer": "", "name": "v0.3.0"}, {"layer": "", "name": "v1.14.0"}, {"layer": "", "name": "v1.15.0"}, {"layer": "", "name": "v1.16.0"}, {"layer": "", "name": "v1.17.0"}, {"layer": "", "name": "v1.18.0"}, {"layer": "", "name": "v1.18.0-cpo.2020.2.26"}, {"layer": "", "name": "v1.18.1"}, {"layer": "", "name": "v1.18.2"}, {"layer": "", "name": "v1.19.0"}, {"layer": "", "name": "v1.19.1"}, {"layer": "", "name": "v1.19.2"}]
```

Reply:
You're correct, the image is still not pushed. I'll update the PR to use 1.19.2 instead.

Reply:
@irozzo-1A Fixed, PTAL.